.gitignore

```diff
@@ -24,6 +24,7 @@ test/e2e/data/infrastructure-docker/v1alpha4/cluster-template*.yaml
 
 # VSCode
 .vscode/
+*.code-workspace
```

Comment:
is this change related to this particular PR?

Reply:
It's not but I'd say it's fine to leave it